server/apps/microservices/src/config/app.config.ts

```diff
@@ -0,0 +1,22 @@
+import { ConfigModuleOptions } from '@nestjs/config';
```

Comment:
having two separate files doing the same thing is not a good thing. We can either move this into the `server/libs` directory where both `server` and `microservices` can share code.

Reply:
I will store `app.config.ts` to `server/libs/config` and import again.

Reply:
Do you wanna add alias `@app/config` in this pull? or I will import relative path like `../../../libs/config/app.config`